pppd/plugins/pppoatm/pppoatm.c

```diff
@@ -92,7 +92,7 @@ static int setdevname_pppoatm(const char *cp, const char **argv, int doit)
 		return 1;
 
 	memcpy(&pvcaddr, &addr, sizeof pvcaddr);
-	strlcpy(devnam, cp, MAXPATHLEN);
+	strlcpy(devnam, cp, sizeof(devname));
```

Comment:
devname not devnam??

Reply:
You are looking at an outdated version of the file, current PR has the change you suggested "devnam"